### PR TITLE
Modify serializers tests to measure space consumption

### PR DESCRIPTION
**Pull request by kjanosz**
Fix RangeBenchmark.
Refactor serializers tests.
Refactor IOStreamPersistor.

### CODE REVIEW

File: src/test/scala/org/scalameter/persistence/GZIPJSONSerializationPersistorTest.scala [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+import org.scalatest.{Matchers, WordSpec}
 
 
-class GZIPJSONSerializationPersistorTest extends FunSuite with PersistorTest with Matchers {
-  test("Should correctly serialize and deserialize history") {
-    afterBenchmark(new GZIPJSONSerializationPersistor) { persistor =>
+class GZIPJSONSerializationPersistorTest extends WordSpec with PersistorTest with Matchers {
+  val persistor = new GZIPJSONSerializationPersistor
+
+  "GZIPJSONSerializationPersistor" should {
```
Comment (axel22) [inline src/test/scala/org/scalameter/persistence/GZIPJSONSerializationPersistorTest.scala]: Could we continue using `FunSuite` and `test("...")` syntax for consistency?

**Comment by kjanosz** [comment]
@axel22: Changed.

Btw, for slurping files I've used `java.nio.file.Files.readAllBytes` which is available from JDK7 - don't know if it can be a problem.

**Comment by axel22** [comment]
Thanks for pointing that out!
Since it's used only in the test, I don't think this should be a problem.

**Comment by kjanosz** [comment]
Also, I still do not have running times of serialization for large benchmarks. 
One large benchmark is still running on my laptop (started about ~3 hours ago) ;-)
Could I post results on mailing lists when they will be ready?

**Comment by axel22** [comment]
LGTM

**Comment by axel22** [comment]
Sure thing - feel free to post them whenever you have them ready!

**Pull request merged**